tests/test_pylint_command.py

```diff
@@ -0,0 +1,35 @@
+import pathlib
+import subprocess
+
+
+def _run_pylint(package_dir_path: pathlib.Path) -> subprocess.CompletedProcess:
+    return subprocess.run(
+        ('pylint', '--score=n', '--load-plugins=pylint_import_requirements', 'module.py'),
+        cwd=package_dir_path,
+        stdout=subprocess.PIPE,
+        check=False,
+    )
```

Comment:
Have you checked out [this document](http://pylint.pycqa.org/en/latest/how_tos/custom_checkers.html#testing-a-checker)? Its seems to be an easier solution than manually running pylint and asserting the output

Reply:
Thanks for the review and the suggestions! :-)

I didn't check the docs :joy:

Did you expect something like this https://github.com/WanzenBug/pylint-import-requirements/blob/2a9fc2830ccbb70da3c6368d64cc2660f2b831a5/tests/test_checker.py ?

Shall we remove the subprocess.run integration tests completely or keep one?

Reply:
I would prefer complete removal.